server/etcdserver/api/etcdhttp/metrics_test.go

```diff
@@ -26,7 +25,6 @@ func (s *fakeStats) StoreStats() []byte  { return nil }
 
 type fakeServerV2 struct {
```

Comment:
If we remove v2 server, then why we keep the fakeServerV2?

Reply:
Haven't looked to deep into this test code, but based on the name I assumed that it tests "/health" endpoint. 
In this PR I removed all "/v2/*" endpoints, however there are still multiple other HTTP endpoints like "/health", "/metrics" etc.  

For `fakeServerV2` I think the name is outdated, remaining endpoints no longer belong to v2 api. We might want to rename them. For example `fakeServerV2` to `fakeHTTPServer`